pallets/auctions/Cargo.toml

```diff
@@ -0,0 +1,55 @@
+[package]
+edition = "2018"
```

Comment:
Make sure that the order or edition, license, name and version is the same as we have in order pallets.